### DISCUSSION TITLE
How do I stop auto indent? Possibly unintended behaviour in python

### DISCUSSION THREAD

**Discussion by YotGun** [zed-industries/zed#11604] [Feature Requests]
Relevant post: [Reddit post](https://www.reddit.com/r/ZedEditor/comments/1cnlkkl/how_do_i_stop_this_autoindent_preferably_in_every/)

[Video / Gif
](https://packaged-media.redd.it/k3zuwt8l2bzc1/pb/m2-res_1080p.mp4?m=DASHPlaylist.mpd&v=1&e=1715266800&s=7f2707a6c274711c6ebe3c88ddea2215f002baa6#t=0)

Hi, I tried coding around in Zed but the most annoying thing to me is the auto indent feature. I can't seem to disable it and there is this 1  specific thing that keeps happening which I believe shouldn't. It auto indents to the last "if" even though I want the "else" to be for the first "if". After typing ":" it automatically shifts it.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
